torch/functional.py

```diff
@@ -167,6 +168,25 @@ def isnan(tensor):
     return tensor != tensor
 
 
+def isinf(tensor):
+    r"""Returns a new tensor with boolean elements representing if each element is `+/-INF` or not.
```

Comment:
Nit: the elements aren't boolean, they're either 0 or 1.

Reply:
I think we should keep it for consistency, anyway it is indeed boolean element right?